SPECS/cert-manager/cert-manager.spec

```diff
@@ -0,0 +1,64 @@
+Name:           cert-manager
+Version:        1.5.3
+Release:        1
+Summary:        Automatically provision and manage TLS certificates in Kubernetes 
+License:        ASL 2.0
+URL:            https://github.com/jetstack/cert-manager
+#Source0:       https://github.com/jetstack/%{name}/archive/refs/tags/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+Source1:        %{name}-%{version}-vendor.tar.gz
```

Comment:
Please add instruction on how to generate (in a repeatable way, so that the hashes don't change for the same version) the "vendor" and "gocache" sources.